tensorflow/core/kernels/data/name_utils.h

```diff
@@ -47,7 +50,8 @@ string DatasetDebugString(const string& dataset_type,
 template <typename... Args>
 string DatasetDebugString(const string& dataset_type, const Args&... args) {
   return DatasetDebugString(
-      dataset_type, {static_cast<const strings::AlphaNum&>(args).Piece()...});
+      dataset_type, "Dataset",
```

Comment:
if you go with prefix, the define "" as `kDefaultDatasetDebugStringPrefix`

Reply:
Done